### DISCUSSION TITLE
Long Time Support (LTS) Project

### DISCUSSION THREAD

**Discussion by Bluemangoo** [LeavesMC/Leaves#270] [Polls]
We are planning commercial long time support(LTS). Maintain outdated Minecraft versions like 1.20.4 1.18, migrate patches back to old versions, and sync latest functional and security updates (if possible) with latest Minecraft version. Current version(1.21) will be included when we move on to next version. This is a paid project, since we are open-source project and need financial support to keep us going. Here we are listening to your idea. And, if you need it, please tell us the Minecraft version you need.

Gratefully, 
LeavesMC Team

---

我们正在考虑付费的长期支持项目：维护旧的 Minecraft 版本，回移植 patch 并同步最新版本的功能和安全更新（如果适用）。当前版本也将在维护周期结束后包括在内。我们真的需要经济上的支持来推动我们的开发。请告诉我们您的想法。如果您需要，请一并告诉我们您需要哪个 Minecraft 版本。感激不尽。

-- LeavesMC 团队

**Comment by LittleChest** [comment]
你应该加版本
```
1.17.1
1.18.2
1.19.4
1.20.4
```
然后加一个`我不需要`选项啊（

**Reply by Bluemangoo** [reply]
版本是要求你回复的，作为参考。不需要知道投票啊。

**Reply by LittleChest** [reply]
> 版本是要求你回复的，作为参考。不需要知道投票啊。

也可

**Comment by Bluemangoo** [comment]
注：team member就别投了。

**Comment by Mooling0602** [comment]
请求长期支持 1.20.4

**Comment by Mooling0602** [comment]
我支持付费，但是不知道你们计划推出怎样的订阅方式？具体的定价是多少？

**Reply by s-yh-china** [reply]
如果推出，我们计划为 版本/月 的付费模式，来激励我们的开发者进行开发。

**Comment by JohnRichard4096** [comment]
那么适用于过期版本的Leaves依然会开源吗？

**Reply by Bluemangoo** [reply]
已经开源的东西不会闭源。

**Reply by JohnRichard4096** [reply]
了解
